wingstructure/structure/polygon.py

```diff
@@ -52,10 +113,46 @@ def calcneutralcenter(outline):
 
 
 def calcprincipalaxis(I_xx, I_yy, I_xy):
+    """Calculates angle of first principal axis
+    
+    Parameters
+    ----------
+    I_xx : float
+        area moment of inertia
+    I_yy : float
+        area moment of inertia
+    I_xy : float
+        area moment of inertia
+    
+    Returns
+    -------
+    float
+        principal axis angle
+    """
+
     return np.arctan2(2*I_xy, I_yy-I_xx)
 
 
 def transform_intertiamoments(I_xx, I_yy, I_xy, φ):
+    """Transform moments of inertia from into rotated coordinate system
+    
+    Parameters
+    ----------
+    I_xx : float
+        area moment of inertia
+    I_yy : float
+        area moment of inertia
+    I_xy : float
+        area moment of inertia
+    φ: float
+        angle to roatet
+    
+    Returns
+    -------
+    tuple of floats
+        moment of inertia in rotated coordinate system
+    """
+
     I_ξξ = (I_xx+I_yy)/2 + (I_yy-I_xx)/2 * np.cos(2*φ) - I_xy * np.sin(2*φ)
```

Comment:
shouldn't we prefer ascii?
Thinking of people who are editing this without a greek alphabet at hand.